.github/docker-images/base-images/ubi8/Dockerfile

```diff
@@ -0,0 +1,52 @@
+FROM redhat/ubi8:latest AS base
```

Comment:
Should be possible to use the same Dockerfile for Amazon Linux and ubi8

Reply:
They're not exactly the same. Some packages need to be installed for amazonlinux and not ubi8 and vice-versa